js/store.js

```diff
@@ -308,9 +312,10 @@ const store = {
           // show third party; it either became visible or is brand new
           thirdParty['isVisible'] = true;
           isNewThirdParty = true;
-          thirdParty['firstPartyHostnames'].forEach((firstPartyHostname) => {
-            this.addFirstPartyLink(firstPartyHostname, target);
-          });
+          thirdParty['firstPartyHostnames']
+            .forEach((firstPartyHostname) => {
+              this.addFirstPartyLink(firstPartyHostname, target);
```

Comment:
Change this to an array of promises and await that as mentioned on the wrong line :D